js/client-chat.js

```diff
@@ -744,7 +744,7 @@
 				if (app.localLadder) return text;
 				if (!target) target = app.user.get('userid');
 
-				var targets = target.split(',');
+				var targets = target.split(',').map($.trim);
```

Comment:
For the record, `map` is an ES5 feature. As is `trim`. We use `$.trim` because client does not have ES5 support. So we can't use `map`, either.